src/extensions/editable/bootstrap-table-editable.js

```diff
@@ -111,6 +111,20 @@
                 return;
             }
 
+            var data = that.getData();
+
+            that.$body.find('a[data-name="' + column.field + '"]').each(function(i, element){
+                var $element = $(element);
+                var $tr = $element.closest('tr');
+                var index = $tr.data('index');
+                var row = data[index];
```

Comment:
I am not sure if I can safely use:
```
var row = data[i];
```
here and get rid of the two lines above...